Content/default/.editorconfig

```diff
@@ -5,4 +5,10 @@ indent_style = space
 indent_size = 4
 charset = utf-8
 trim_trailing_whitespace = true
-insert_final_newline = false
\ No newline at end of file
+insert_final_newline = false
+
+[*.fs]
+fsharp_space_before_colon=true
```

Comment:
This is not the default option, right? 
If not then why include it - personal preference?
In general I believe we should start with default configuration

Reply:
This was to match the existing style I found. Is not the default indeed.
Remove?